packages/wallet/src/redux/sagas/message-listener.ts

```diff
@@ -94,6 +108,59 @@ export function* messageListener() {
   }
 }
 
+function* validateTransitionForCommitments(signedCommitments: SignedCommitment[]) {
+  const channelId = getChannelId(signedCommitments[0].commitment);
+  // If we're receiving the first commitment there's nothing stored to validate against
+  if (signedCommitments[0].commitment.turnNum > 0) {
```

Comment:
I think we would want to ask directly if we have something stored to validate against. If B sends [{turnNum: 0}, {turnNum: 1}]` to A, then we wouldn't enter this block.

Reply:
Updated